charts/tfy-notebook-controller/values.yaml

```
@@ -91,31 +91,8 @@ oauth:
 sdsServer:
   ## @param sdsServer.image image for sds-server
   image: "public.ecr.aws/truefoundrycloud/sds-server:cb6f4be9a5e342ad7a3ba4eb72bcc158501500d8"
+  ## @param sdsServer.affinity [object] Node affinity for sds-server
   affinity:
```

Don't you have to remove this field?

---

i have removed the value but good to have since we might need it for aws

---

If the template already supports the key, why must we have it incomplete in the values file?

```suggestion
  affinity: {}
```